content/docs/studio/python-api.md

```diff
@@ -0,0 +1,35 @@
+# Python API
+
+The purpose of Studio Python API is to give programmatic access to information
+in Studio and executing actions in it.
+
+We provide Studio Python API as a part of DVC python package. To use it, you
+need to [install](/doc/install) DVC with `pip` or `conda`.
+
+This reference provides the details about the functions in the `dvc.studio`
+module, which can be loaded in any regular way, for example:
+
+```py
+import dvc.studio
+```
+
+To use this API, you need to
+[generate](/doc/studio/user-guide/account-management#studio-access-token) and
+[set up](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots#set-up-an-access-token)
+Studio access token.
+
+## Download model
+
+Download model binaries for a model from Model Registry. Requires the model to
+be stored with DVC with s3 or azure remote. Note, that you need to
```

Comment:
Just curious, why isn't GCS supported?

Reply:
Answered in https://github.com/iterative/studio/pull/6338#issuecomment-1623808358